rapids-cmake/cpm/versions.json

```diff
@@ -16,7 +16,7 @@
       "version": "2.8.3",
```

Comment:
Should this be 2.8.4 now? Or should will we bump this again after an official CCCL release?

Reply:
We will bump this again once a tag comes out. We know we need this fix, with or without a tag.

Reply:
OK sounds good.